src/odemis/gui/util/__init__.py

```diff
@@ -282,3 +283,17 @@ class AttrDict(dict):
     def __init__(self, *args, **kwargs):
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+
+def enable_cryo_tab_on_stage_position(tab, stage, pos):
```

Comment:
Let's name this function something a little bit more specific, like `enable_tab_on_imaging_position`. Alternatively, we could accept an extra parameter "target", which would be typically `IMAGING`, but could be another position, and name it  `enable_tab_on_stage_position`.